src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -3073,11 +3080,11 @@ private void WriteToServerInternalRestContinuedAsync(BulkCopySimpleResultSet int
 
                 try
                 {
-                    CleanUpStateObjectOnError();
+                    CleanUpStateObject(isCancelRequested: true);
                 }
                 catch (Exception cleanupEx)
                 {
-                    Debug.Fail("Unexpected exception during CleanUpstateObjectOnError (ignored)", cleanupEx.ToString());
+                    Debug.Fail("Unexpected exception during CleanUpstateObject (ignored)", cleanupEx.ToString());
```

Comment:
this can be changed to `Debug.Fail("Unexpected exception during CleanUpstateObject (ignored)", cleanupEx.ToString());
```suggestion
                    Debug.Fail($"Unexpected exception during {nameof(CleanUpstateObject)} (ignored)", cleanupEx.ToString());
```